src/Components/Components/src/EventCallbackFactoryBinderExtensions.cs

```diff
@@ -346,14 +346,16 @@ private static bool ConvertToNullableDateTimeCore(object obj, out DateTime? valu
         /// <param name="receiver"></param>
         /// <param name="setter"></param>
         /// <param name="existingValue"></param>
+        /// <param name="culture"></param>
         /// <returns></returns>
         public static EventCallback<UIChangeEventArgs> CreateBinder(
             this EventCallbackFactory factory,
             object receiver,
             Action<string> setter,
-            string existingValue)
+            string existingValue,
+            CultureInfo culture = null)
```

Comment:
I made these methods accept a culture even where it doesn't do anything (for consistency). These are targeted by codegen + overload resolution, so the error experience won't be good if you specify a culture, but we don't accept one.